functions-methods.js

```diff
@@ -30,6 +62,26 @@
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
 // checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
 
+function checkEmailValidity(emailadres) {
+
+    const containsAt = emailadres.includes("@");
+    const containsComma = emailadres.includes(",");
+    const containsDot = emailadres.lastIndexOf(".");
+    const containsDotAtEnd = containsDot !== emailadres.length -1;
+
+    if(containsAt && !containsComma && containsDotAtEnd) {
+        return 'true';
+    } else {
+        return 'false';
+    }
```

Comment:
Je kan de uitkomst van de expressie meteen teruggeven:
```suggestion
    return containsAt && !containsComma && containsDotAtEnd;
```